app/ai/voice/llm/types.py

```
@@ -59,6 +59,30 @@ class RealtimeConfig(BaseModel):
+        None,
+        description="Gemini 3.x Live reasoning level: 'minimal', 'low', "
+        "'medium', or 'high' (3.1-flash-live defaults to 'minimal' for lowest "
+        "latency). Only applied when set; otherwise the model default applies. "
+        "Gemini 2.5 native-audio uses thinking_budget and ignores this. "
+        "Not used by OpenAI/xAI/Azure.",
+    )
+    silence_duration_ms: Optional[int] = Field(
+        None,
+        description="Gemini Live server-side VAD end-of-speech threshold in "
+        "milliseconds (how long a pause ends the user's turn; 3.1-flash-live "
+        "recommends 500–800ms). Only applied when set; otherwise Gemini's "
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)